### DISCUSSION TITLE
Tell Robot to clean a certain room

### DISCUSSION THREAD

**Discussion by Vippis2000** [Tasshack/dreame-vacuum#599] [General]
Hi @Tasshack 

How can I tell the robot to clean a certain room as in Dreame App?

I use dreame-vacuum 2.x DEV with Dreame App Support.

I also use hass adapter with IOBroker as I use hass only for your integration and mainly IOBroker for my other devices and visualization.

I can see the "send_command" datapoint in vacuum folder in IOBroker.

Whats the command to clean segment 1?

**Comment by Tasshack** [comment]
You are asking the most basic thing that can be done from the integration.
Have you checked the docs?

**Reply by Vippis2000** [reply]
I've read the docs for sure...

From IOBroker I don't see the entity clean_segment.

Is it possible to give the clean_segment via send_command ?

**Reply by Tasshack** [reply]
I don't recommend using send_command for any reason because integration should be aware of every request to set some states.
I don't know how IOBroker works but it should have access to [clean_segment](https://github.com/Tasshack/dreame-vacuum/blob/master/docs/services.md#dreame_vacuumvacuum_clean_segment) service as is on send_command.

**Reply by Vippis2000** [reply]
No only entities are visible via HASS Adapter for IOBroker.

I have helped myself by triggering Service in HASS via MQTT trigger listening to a certain topic to start clean_segment service.
